README.rst

```diff
@@ -0,0 +1,122 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-st7789/badge/?version=latest
+    :target: https://circuitpython.readthedocs.io/projects/st7789/en/latest/
+    :alt: Documentation Status
+
+.. image:: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+.. image:: https://travis-ci.com/adafruit/Adafruit_CircuitPython_ST7789.svg?branch=master
+    :target: https://travis-ci.com/adafruit/Adafruit_CircuitPython_ST7789
+    :alt: Build Status
+
+displayio driver for ST7789 TFT-LCD displays.
+
+Dependencies
+=============
+This driver depends on:
+
+* `Adafruit CircuitPython 4.0.0-beta.0+ <https://github.com/adafruit/circuitpython>`_
+
+Please ensure all dependencies are available on the CircuitPython filesystem.
+This is easily achieved by downloading
+`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+
+Usage Example
+=============
+
+.. code-block:: python
+
+    import board
+    import displayio
+    from adafruit_st7789 import ST7789
+
+    spi = board.SPI()
```

Comment:
Can you configure the SPI to higher speed than the default? It's painfully slow. I think it would be best to configure it to display's maximum.

Reply:
Sure. It looks like that's been added since I made this. Could you remind me of the syntax to change it and the speed you think would work better? Were you thinking 24MHz?

Reply:
I found something on discord and used that to set it to 24MHz.